dcc-portal-ui/app/scripts/common/styles/_header.scss

```diff
@@ -432,13 +431,13 @@ header {
           size: 4rem;
           weight: 300;
           family: $sans-stack;
-        /*@include breakpoint($break-tablet) {
-          line-height: 7rem;
-          font: {
-            size: 4rem;
-            weight: 300;
-            family: $sans-stack;
-          } */
+        // /*@include breakpoint($break-tablet) {
```

Comment:
going to remove these comments before PR is sent for merging, just wanted to see what the jar would build to

/**/ apparently sometimes break the build with node-sass